Assets/MixedRealityToolkit.Services/CameraSystem/MixedRealityCameraSystem.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.CameraSystem
+{
+    /// <summary>
+    /// The Camera system controls the settings of the main camera.
+    /// </summary>
+    public class MixedRealityCameraSystem : BaseCoreSystem, IMixedRealityCameraSystem
+    {
+        public MixedRealityCameraSystem(
+            IMixedRealityServiceRegistrar registrar,
+            BaseMixedRealityProfile profile = null) : base(registrar, profile)
+        {
+            cameraProfile = (MixedRealityCameraProfile)profile;
+        }
+
+        private MixedRealityCameraProfile cameraProfile;
+
+        /// <inheritdoc />
+        public uint SourceId { get; } = 0;
+
+        /// <inheritdoc />
+        public string SourceName { get; } = "Mixed Reality Camera System";
+
+        /// <inheritdoc />
+        public override void Initialize()
+        {
+            if (cameraProfile.IsOpaque)
+            {
+                ApplySettingsForOpaqueDisplay();
+            }
+            else
+            {
+                ApplySettingsForTransparentDisplay();
+            }
+        }
+
+        /// <inheritdoc />
+        public override void Update()
+        {
+            if (cameraProfile.IsOpaque)
```

Comment:
We should probably check if this has changed, instead of setting it each update.